.github/workflows/build.yml

```diff
@@ -117,8 +117,13 @@ jobs:
       DOCKER_BUILDKIT: 1
 
     strategy:
+      max-parallel: 12
       matrix:
-        boards: [arm-01, arm-02, arm-03, arm-04, arm-05, arm-06, arm-07, arm-08, arm-09, arm-10, arm-11, arm-12, arm-13, other, risc-v-01, risc-v-02, sim-01, sim-02, xtensa-01, xtensa-02]
+        boards: [
+          arm-01, other, risc-v-01, sim-01, xtensa-01,
```

Comment:
This does not seem to limit runners just change the execution order? Does this improve execution because different platforms are started first and later executions are known to fail?

Reply:
`max-parallel: 12` will limit to 12 runners. Previously it was unlimited. We limit the runners because of ASF Policy: https://lupyuen.github.io/articles/ci#appendix-phase-1-of-ci-upgrade

Reply:
Thanks @lupyuen 

I did a rebase to see if this can fix the CI error fails on macOS jobs.